dev/cloud/teardown.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Magento\Framework\App\Bootstrap;
+require '/app/app/bootstrap.php';
+
+$bootstrap = Bootstrap::create(BP, $_SERVER);
+$obj = $bootstrap->getObjectManager();
```

Comment:
```suggestion
$objectManager = $bootstrap->getObjectManager();
```

Reply:
This code is simple and not that long, so people may not confound the purpose of that variable here (a manager of things for a thing), but I do believe it's a good practice to always remove as much ambiguity as possible, even if the name of the variable is a bit longer.